package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "plop-action-eslint",
   "description": "Use ESLint to format generated code with plop.js",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Hi, thanks for suggesting changing this as well. As I use Lerna in this repo to release a new version of the package this version is not required actually.

```suggestion
  "version": "0.0.1",
```